MySQL error

Hi, I have such a problem that after connecting the plug to the database I get this error

> [08:24:23] [Server thread/ERROR]: Error occurred while enabling PreciousStones v1.16.1.9 (Is it up to date?)
java.lang.IllegalStateException: Unable to initialise database
	at net.sacredlabyrinth.Phaed.PreciousStones.storage.StorageManager.<init>(StorageManager.java:76) ~[?:?]
	at net.sacredlabyrinth.Phaed.PreciousStones.PreciousStones.onEnable(PreciousStones.java:148) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugin(CraftServer.java:501) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugins(CraftServer.java:415) ~[patched_1.16.3.jar:git-Paper-247]
	at net.minecraft.server.v1_16_R2.MinecraftServer.loadWorld(MinecraftServer.java:468) ~[patched_1.16.3.jar:git-Paper-247]
	at net.minecraft.server.v1_16_R2.DedicatedServer.init(DedicatedServer.java:237) ~[patched_1.16.3.jar:git-Paper-247]
	at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:939) ~[patched_1.16.3.jar:git-Paper-247]
	at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.3.jar:git-Paper-247]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Specified key was too long; max key length is 767 bytes
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_265]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_265]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_265]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_265]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:403) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.Util.getInstance(Util.java:386) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2524) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2675) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2465) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1915) ~[patched_1.16.3.jar:git-Paper-247]
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1254) ~[patched_1.16.3.jar:git-Paper-247]
	at net.sacredlabyrinth.Phaed.PreciousStones.shaded.com.zaxxer.hikari.pool.ProxyPreparedStatement.execute(ProxyPreparedStatement.java:44) ~[?:?]
	at net.sacredlabyrinth.Phaed.PreciousStones.shaded.com.zaxxer.hikari.pool.HikariProxyPreparedStatement.execute(HikariProxyPreparedStatement.java) ~[?:?]
	at net.sacredlabyrinth.Phaed.PreciousStones.storage.TableCreator.execute(TableCreator.java:40) ~[?:?]
	at net.sacredlabyrinth.Phaed.PreciousStones.storage.TableCreator.addIndexes(TableCreator.java:223) ~[?:?]
	at net.sacredlabyrinth.Phaed.PreciousStones.storage.TableCreator.createSnitches(TableCreator.java:197) ~[?:?]
	at net.sacredlabyrinth.Phaed.PreciousStones.storage.TableCreator.createTables(TableCreator.java:30) ~[?:?]
	at net.sacredlabyrinth.Phaed.PreciousStones.storage.StorageManager.initiateDB(StorageManager.java:96) ~[?:?]
	at net.sacredlabyrinth.Phaed.PreciousStones.storage.StorageManager.<init>(StorageManager.java:74) ~[?:?]
	... 11 more
[08:24:23] [Server thread/INFO]: [PreciousStones] Disabling PreciousStones v1.16.1.9
[08:24:23] [Server thread/INFO]: [PreciousStones] Shutting Down: Cancelling all tasks...
[08:24:23] [Server thread/INFO]: [PreciousStones] Shutting Down: Saving all pending data...
[08:24:23] [Server thread/ERROR]: Error occurred while disabling PreciousStones v1.16.1.9 (Is it up to date?)
java.lang.NullPointerException: null
	at net.sacredlabyrinth.Phaed.PreciousStones.PreciousStones.onDisable(PreciousStones.java:225) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:415) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:516) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:384) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugin(CraftServer.java:501) ~[patched_1.16.3.jar:git-Paper-247]
	at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugins(CraftServer.java:415) ~[patched_1.16.3.jar:git-Paper-247]
	at net.minecraft.server.v1_16_R2.MinecraftServer.loadWorld(MinecraftServer.java:468) ~[patched_1.16.3.jar:git-Paper-247]
	at net.minecraft.server.v1_16_R2.DedicatedServer.init(DedicatedServer.java:237) ~[patched_1.16.3.jar:git-Paper-247]
	at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:939) ~[patched_1.16.3.jar:git-Paper-247]
	at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.3.jar:git-Paper-247]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]

And when you join and leave the game, this problem arises
https://prnt.sc/v7pgxf
https://prnt.sc/v7pi4y

--- a/src/main/java/net/sacredlabyrinth/Phaed/PreciousStones/storage/TableCreator.java
+++ b/src/main/java/net/sacredlabyrinth/Phaed/PreciousStones/storage/TableCreator.java
@@ -169,7 +169,7 @@ class TableCreator implements AutoCloseable {
     private void createPlayers() throws SQLException {
         execute("CREATE TABLE IF NOT EXISTS `pstone_players` ("
                 + "`id` bigint(20), "
-                + "`uuid` varchar(255) default NULL, "
+                + "`uuid` varchar(36) default NULL, "
                 + "`player_name` varchar(16) NOT NULL, "
                 + "`last_seen` bigint(20) default NULL, "
                 + "flags TEXT default NULL, "

